src/tsdf/write_binary.py

```diff
@@ -48,5 +75,6 @@ def write_binary_file(
     data.tofile(path)
     metadata.update(_get_metadata_from_ndarray(data))
     metadata.update({"file_name": file_name})
+    #TODO: update file_dir_path?
```

Comment:
`file_dir` is not a field in the metadata, it is just a parameter we save on the level of the TSDF object, so we can automatically read and write files from the file system (e.g., the binaries described in the tsdf metadata). If this is not a correct representation, we could discuss it offline.